packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -23,13 +23,15 @@ import { ApiClient } from '../api';
 import { timeOperation } from '../metrics';
 import { readGraphObjectFile } from '../storage/FileSystemGraphObjectStore/indices';
 import { FlushedGraphObjectData } from '../storage/types';
+import { retry } from '@lifeomic/attempt';
 
 export { synchronizationApiError };
 import { createEventPublishingQueue } from './events';
+import { AxiosInstance } from 'axios';
 export { createEventPublishingQueue } from './events';
 
 const UPLOAD_BATCH_SIZE = 250;
-const UPLOAD_CONCURRENCY = 2;
+const UPLOAD_CONCURRENCY = 6;
```

Comment:
Is there an optimal concurrency here? I would have _no_ idea how to determine what to use.

Reply:
Not particularly. Increasing and plan to monitor in the coming days.

Reply:
It would be amazing someday to have our APIs answer concurrency headers so that clients could scale up and down as the system is able to support load. 😎